superset-frontend/src/cccs-viz/plugins/plugin-chart-at-a-glance-user-id/src/AtAGlanceUserID.tsx

```diff
@@ -0,0 +1,268 @@
+import React, { useState } from 'react';
+import { RiGlobalFill } from 'react-icons/ri';
+import { QueryFormData } from '@superset-ui/core';
+import Collapse from 'src/components/Collapse';
+
+type DataManager = {
+  formData: QueryFormData;
+  data: any[];
+  isInit: boolean;
+  isLoading: boolean;
+  isError: boolean;
+};
+
+// TODO: Fix this so links between dashboards will work again
```

Comment:
Is there a ticket for this that we could reference here, just to make sure we don't forget about it?

Reply:
https://cccs.atlassian.net/browse/CLDN-1258

Reply:
done